lib/ignore/cache.go

```diff
@@ -21,7 +21,7 @@ type cache struct {
 
 type cacheEntry struct {
 	result Result
-	access time.Time
+	access int64 // Unix nanosecond count. Sufficient until the year 2262.
```

Comment:
changing the order of these should also reduce the size of the struct a fair bit I think

Reply:
It goes in a map, where it's rounded up to the alignment size, so it would only matter on 32-bit platforms.

Reply:
Are you sure? I thought map values were stored packed.

Reply:
I don't have a reference to the runtime code handy, but some quick benchmarking suggests that a map with values of type `struct { x uint64; y byte }` is as big as one of `struct { x, y uint64 }`.